server/src/main/java/org/opensearch/index/shard/RemoteStoreRefreshListener.java

```diff
@@ -193,6 +202,8 @@ private synchronized void syncSegments(boolean isRetry) {
                                 OnSuccessfulSegmentsSync();
```

Comment:
I understand that this is not related to this PR: But can you please change this to `onSuccessfulSegmentsSync`? (`o` in small case)